base/src/main/java/io/spine/validate/MessageFieldValidator.java

```diff
@@ -131,6 +131,14 @@ private static boolean isTimeInvalid(Timestamp timeToCheck, Time whenExpected, T
         return isInvalid;
     }
 
+    private static boolean isLaterThan(Timestamp t1, Timestamp t2) {
```

Comment:
Please have this method in `Timestamps` utility file.

Please also have `fromMillis()` also in this utility class.